ceph/ceph_cluster.yml

```diff
@@ -6,25 +6,29 @@ parameters:
  prefix: ceph
  numcpus: 2
  memory: 4096
- image: centos8
+ image: fedora32
  notify: false
  admin_password: password
  disks:
  - 150
- - 150
+ - 15
 
 {% for number in range(0, nodes) %}
 {{ prefix }}-node-0{{ number }}:
  image: {{ image }}
  numcpus: {{ numcpus }}
  memory: {{ memory }}
+ reserveip: true
```

Comment:
What's this for? What I added here instead was a `reservehost: true` and it simply added the `ceph-host-xx` to the /etc/hosts (it requires password entry while starting this though).

Reply:
https://kcli.readthedocs.io/en/latest/index.html?highlight=reserveip#ip-dns-and-host-reservations

I want to assign automatically ip addresses and create DNS entries in the libvirt virtual network. . And if i sues the same name for the vms, reuse again the same ips